tests/components/unifi/test_device_tracker.py

```diff
@@ -203,6 +203,16 @@ def mock_controller(hass):
     device_1 = hass.states.get("device_tracker.device_1")
     assert device_1.state == "home"
 
+    device_1_copy = copy(DEVICE_1)
+    device_1_copy["disabled"] = True
+    mock_controller.mock_client_responses.append({})
+    mock_controller.mock_device_responses.append([device_1_copy])
+    await mock_controller.async_update()
+    await hass.async_block_till_done()
+
+    device_1 = hass.states.get("device_tracker.device_1")
+    assert device_1.state == "unavailable"
```

Comment:
Ideally, we'd use the constant `STATE_UNAVAILABLE`

Reply:
I will correct that with my next PR @andrewsayre. Thanks!